src/main/java/ru/yandex/practicum/filmorate/controllers/ErrorHandler.java

```diff
@@ -0,0 +1,55 @@
+package ru.yandex.practicum.filmorate.controllers;
+
+import com.sun.jdi.InternalException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import ru.yandex.practicum.filmorate.exceptions.FilmException;
+import ru.yandex.practicum.filmorate.exceptions.UserException;
+import ru.yandex.practicum.filmorate.exceptions.ValidationException;
+
+@Slf4j
+@RestControllerAdvice
+public class ErrorHandler {
+    @ExceptionHandler
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public ErrorResponse handlerValidationException(final ValidationException e) {
+        log.warn("Ошибка проверки данных " + e.getMessage());
+        return new ErrorResponse(e.getMessage());
+    }
+
+    @ExceptionHandler
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    public ErrorResponse handlerUserException(final UserException e) {
+        log.warn("Ошибка при работе с пользователем", e.getMessage());
```

Comment:
Не хватает места для параметра `{}`
```suggestion
        log.warn("Ошибка при работе с пользователем {}", e.getMessage());
```
И всех всех логах ниже